functions/src/schedule-generator/speakers-sessions-map.js

```diff
@@ -0,0 +1,46 @@
+function sessionsSpeakersMap(sessionsRaw, speakersRaw) {
+    let sessions = {};
+    let speakers = {};
+
+    Object.keys(sessionsRaw).forEach((sessionId) => {
+        const currentSession = sessionsRaw[sessionId]
+        const sessionSpeakers = [];
+        const mainTag = currentSession.tags ? currentSession.tags[0] : 'General';
+
+        currentSession.speakers &&
+            currentSession.speakers.forEach(speakerId => {
+                if (!speakersRaw[speakerId]) return;
+                sessionSpeakers.push(Object.assign({}, { id: speakerId }, speakersRaw[speakerId]));
```

Comment:
you can use spread operator in functions, code should looks better after this :)